sandbox/grist/schema.py

```diff
@@ -261,6 +261,7 @@ def schema_create_actions():
       make_column("label", "Text"),
       make_column("memo", "Text"),
       make_column("enabled", "Bool"),
+      make_column("columnRefList", "RefList:_grist_Tables_column")
```

Comment:
Maybe for max consistency with `isReadyColRef` it could be something like `watchedColRefList`.